### ISSUE TITLE
[Feature] SenseNova-U1.5-8B-MoT

### ISSUE THREAD

**Issue by GreenShadows** [leejet/stable-diffusion.cpp#1841]
### Feature Summary

SenseNova-U1.5 is a natively unified multimodal model built on NEO-unify, featuring new patch encoding and decoding layers.

### Detailed Description

<img width="2956" height="1397" alt="Image" src="https://github.com/user-attachments/assets/c47fc3f9-d09b-4cb6-824f-56cf1f9ac29f" />

<img width="1080" height="1947" alt="Image" src="https://github.com/user-attachments/assets/22ab3ec0-aa85-434a-88aa-7be9f9a8a467" />

<img width="856" height="1280" alt="Image" src="https://github.com/user-attachments/assets/88ac9cdc-296b-4fde-ab17-11ca6c5c2c2f" />

"SenseNova-U1's original MLP head reconstructs each RGB patch independently. At high resolutions, this can expose token boundaries as grid patterns, seams, or texture discontinuities. SenseNova-U1.5 instead adopts progressive spatial reconstruction via ConvDecoder: visual tokens are reshaped into a two-dimensional feature grid and upsampled through multiple Pixel Shuffle stages, with intermediate 3×3 convolutions enabling neighboring patches to interact and merge into a more continuous image.

Qualitative results show that this design substantially suppresses grid-like artifacts throughout pre-training. It also improves robustness during downstream adaptation, reducing the likelihood that similar artifacts re-emerge when fine-tuning on new domains."

HF: https://huggingface.co/sensenova/SenseNova-U1.5-8B-MoT-Preview

GitHub: https://github.com/OpenSenseNova/SenseNova-U1

**Comment by Green-Sky** [comment]
It has been surprising to me to not see more MoT models.
